### PR TITLE
File Checksums

### PR DESCRIPTION
**Pull request by pgierz**
First steps for file checksums. See https://app.clickup.com/t/861n41hpy

### CODE REVIEW

File: src/esm_runscripts/compute.py [hunk 1]
```diff
@@ -1,31 +1,52 @@
...
 import esm_calendar
 import esm_parser
 import esm_runscripts
+import esm_tools
 
 from .batch_system import batch_system
 from .filelists import copy_files, log_used_files
 from .helpers import end_it_all, evaluate, write_to_log
 from .namelists import Namelist
-from loguru import logger
```
Comment (pgierz) [inline src/esm_runscripts/compute.py:22]: Only "new" thing is `hashlib`, the rest is just `isort` doing its thing. At some point I really should show the team [`pre-commit`](https://pre-commit.com)....

File: src/esm_runscripts/compute.py [hunk 2]
```diff
@@ -300,6 +321,7 @@ def _write_finalized_config(config):
     ----------
     config : esm-tools config object
     """
+
```
Comment (pgierz) [inline src/esm_runscripts/compute.py:324]: All changes below this line are just `black` running in the background.

**Comment by mandresm** [comment]
Isn't this duplicated here #972?

**Comment by pgierz** [comment]
....yes it is. Sorry, forgot about that one. 

We should start merging, there are too many half finished things floating around. I'm starting to lose track. We dont want to enter the fesom refactoring world.

**Pull request closed without merging**